lms/static/js/student_account/views/LoginView.js

```diff
@@ -212,6 +212,25 @@
                     msg = gettext('An error has occurred. Check your Internet connection and try again.');
                 } else if (error.status === 500) {
                     msg = gettext('An error has occurred. Try refreshing the page, or check your Internet connection.'); // eslint-disable-line max-len
+                } else if (error.responseJSON !== undefined && error.responseJSON.error_code === 'inactive-user') {
+                    msg = HtmlUtils.interpolateHtml(
+                    gettext('In order to sign in, you need to activate your account.{line_break}{line_break}' +
+                            'We just sent an activation link to {strong_start} {email} {strong_end}. If ' +
+                            ' you do not receive an email, check your spam folders or ' +
+                            ' {anchorStart}contact {platform_name} Support{anchorEnd}.'),
```

Comment:
nit: Add space either at the end of the line or at the start, adding on both sides will add two spaces between two words.